docs/create_deploy.md

```diff
@@ -1,107 +1,114 @@
 Creating and Deploying a Contract
 ================================
 
-There are 3 type of environments Remix can be plugged to:
-`Javascript VM`, `Injected provider`, or `Web3 provider`. (for details see [Running transactions](https://remix-ide.readthedocs.io/en/latest/run.html))
-
-Both `Web3 provider` and `Injected provider` require the use of an
-external tool.
-
-The external tool for `Web3 provider` is an Ethereum node and for
-`Injected provider` Metamask.
-
-The `JavaScript VM` mode is convenient because each execution runs in
-your browser and you don't need any other software or Ethereum node to run it. 
-
-So, it is the easiest test environment - **no setup required!**
-
-But keep in mind that reloading the browser when you are in the Javascript VM will restart Remix in an empty state.
-
-For performance purposes ( which is to say - for testing in an environment that is closest to the mainnet), it might also be better to use an external node.
-
-Selecting the VM mode
----------------------
-
-Make sure the VM mode is selected. All accounts displayed in `Accounts`
-should have 100 ether.
+This page contains the process of creating a contract, compiling it, deploying and then interacting with it.
 
 Sample contract
 ---------------
+This contract is very basic. The goal is to quickly start to create and
+to interact with a sample contract.
+
+![](images/a-file-explorer-new-file2.png)
+
+Go to the File Explorer, create a new file, name it and in the editor paste the contract below.
 
 ``` 
-pragma solidity ^0.5.1;
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.7.6;
 
 contract testContract {
 
-    uint value;
+    uint256 value;
 
-    constructor (uint _p) public {
+    constructor (uint256 _p) {
         value = _p;
     }
 
-    function setP(uint _n) payable public {
+    function setP(uint256 _n) payable public {
         value = _n;
     }
 
-    function setNP(uint _n) public {
+    function setNP(uint256 _n) public {
         value = _n;
     }
 
-    function get () view public returns (uint) {
+    function get () view public returns (uint256) {
         return value;
     }
 }
 
 ```
 
-This contract is very basic. The goal is to quickly start to create and
-to interact with a sample contract.
+Compile the file
+----------------
+With the contract above as the active tab in the Editor, compile the contract.  
 
-Deploying an instance
+**For More Info** see the docs on the ([Solidity Compiler](compile.html)).
+
+Deploy the file
+----------------
+Go to the **Deploy & Run** plugin.
+
+There are 3 type of environments Remix can be plugged to:
+* Javascript VM
+* Injected Web3
+* Web3 Provider
+
+ (For details see [Running transactions](https://remix-ide.readthedocs.io/en/latest/run.html))
+
+Both **Injected Web3** and **Web3 Provider** require the use of an external tool.
+
+The external tool for `Injected provider` is Metamask and for `Web3 provider` is an Ethereum node itself.
```

Comment:
We can frame it like this:
"Example for external tool for `Injected provider` can be Metamask and for `Web3 provider`, it can be a Truffle Ganache-CLI, Hardhat node or an Ethereum node itself"